2023-review.md

```diff
@@ -0,0 +1,164 @@
+| [Symfony & Doctrine](https://symfonycasts.com/screencast/symfony-doctrine)       | 155          |
+| [EasyAdminBundle](https://symfonycasts.com/screencast/easyadminbundle)           | 147          |
+| [Symfony Security](https://symfonycasts.com/screencast/symfony-security)         | 121          |
+| [ApiPlatform Ep2](https://symfonycasts.com/screencast/api-platform-security)     | 102          |
+| [Stimulus](https://symfonycasts.com/screencast/stimulus)                         | 94           |
+| [Messenger](https://symfonycasts.com/screencast/messenger)                       | 88           |
+| [Symfony 5](https://symfonycasts.com/screencast/symfony5)                        | 73           |
+
+If you zoom into the final 3 months, not surprisingly LAST Stack and 
+API Platform episode 3 are 6th and 8th.
+
+How long does it take to make a tutorial? Great question! The biggest variance
+depends on research. Yup, sometimes I have a lot to learn before hitting record!
+
+***TIP
+* `wh` - "work hours" - the number of hours of work spent
+* `vh` - "video hour" - the number of hours of video produced
+
+The times below are *my* hours: they don't take into account editing and
+other important things I have help with.
+***
+
+* [30 Days of LAST Stack](https://symfonycasts.com/screencast/last-stack)
+  `126 hours` spent / `4.1 hour` tutorial = `30.7` wh/vh
+* [API Platform Ep3](https://symfonycasts.com/screencast/api-platform-extending)
+  `97 hours` spent / `3.7 hour` tutorial = `26.2` wh/vh
+* [API Platform Ep2](https://symfonycasts.com/screencast/api-platform-security)
+  `83 hours` spent / `3.7 hour` tutorial = `22.4` wh/vh
+* [API Platform Ep1](https://symfonycasts.com/screencast/api-platform)
+  `58 hours` spent / `2.9 hour` tutorial = `20.4` wh/vh
+* [AssetMapper](https://symfonycasts.com/screencast/asset-mapper)
+  `37 hours` spent / `2.1 hour` tutorial = `17.6` wh/vh
+* [Integration Testing](https://symfonycasts.com/screencast/phpunit-integration)
+  `26.5 hours` spent / `1.0 hour` tutorial = `26.5` wh/vh
+* [Doctrine Queries](https://symfonycasts.com/screencast/doctrine-queries)
+  `25 hours` spent / `1.4 hour` tutorial = `17.8` wh/vh
+
+The key is ``wh/vh``: how much work goes into each hour of video produced.
+I have a few ideas to improve this:
+
+* **New tutorial authoring tool**. We use a fantastic, internal tool called
+  TutsHero to help us build the code for each tutorial. But it's time to
+  modernize that and better integrate it with the recording process.
+
+* **AI in Script Editing**. Fun fact: I first record rough audio while recording
+  the video. We use AI to transcribe that. Then we manually clean up the
+  script, reformatting code (`foo arow bar` becomes `$foo->bar()`), adding
+  `ticks` around code, and rephrasing. Finally, I record this. The script
+  editing this takes a significant amount of time and I think AI can help
+  automate this.
```

Comment:
Should we mention adding our dynamic code blocks too? That still takes some time to add them 😇